editor/inserter/menu.js

```diff
@@ -57,6 +72,10 @@ class InserterMenu extends Component {
 		} );
 	}
 
+	speakAssertive( message ) {
```

Comment:
Yeah, I've just copied from `FormTokenField` but I see it's used more the one time there,